hr_attendance_report_theoretical_time/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2017-2019 Tecnativa - Pedro M. Baeza
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Theoretical vs Attended Time Analysis",
+    "version": "16.0.1.0.0",
+    "category": "Human Resources",
+    "website": "https://github.com/OCA/hr-attendance",
+    "author": "Tecnativa, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "installable": True,
+    "depends": ["hr_attendance", "hr_holidays_public", "test_resource"],
```

Comment:
`test_resource` is not a dependency that I like, but I understand that it is needed now.

Reply:
This is not correct. You can't depend on a test module. Please explain the reasons, and if needed, then you should create an extra module `test_hr_attendance_report_theoretical_time` for that.

Reply:
Hello @pedrobaeza , the reason behind is `TestResourceCommon` has been moved to a dedicated module https://github.com/OCA/hr-attendance/pull/121/commits/df6b8e894bcabe5d081b8350167ed3600fa608a7#diff-d4983130c97220a392bdca5e411cb70875037e6e877ecc3f5f79f5e43707c596R9

Reply:
I will create a new test module to manage that change, thanks for your advise

Reply:
Or you can just simply duplicate the code you need for your test in the `setUpClass`. Check the number of lines you need, and probably it will be the best option.